tests/integration/test_charm.py

```diff
@@ -314,3 +320,31 @@ def juju_reports_one_primary(unit_messages):
 
     # cleanup, remove killed unit
     await ops_test.model.destroy_unit(target_unit)
+
+
+async def test_audit_log(ops_test: OpsTest) -> None:
```

Comment:
will there be other audit log tests in the future ? if so lets make this its own file

Reply:
I don't know, I prefer to keep it here for this PR